app/models/user.rb

```diff
@@ -3,11 +3,16 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: :user_id
   has_many :likes, foreign_key: :user_id
 
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As part of this activity's requirements, we want a user to be asked for confirmation of email during the registration process, therefore let's ensure to enable the `:confirmable` option.